examples/schema/mapping_set_confidence.sssom.tsv

```diff
@@ -0,0 +1,15 @@
+#curie_map:

```

Comment:
Given that the `mapping_set_confidence` slot did not exist in SSSOM 1.0 and will be an addition in SSSOM 1.1, this set is only compliant with version 1.1 and therefore should have a `sssom_version: "1.1"` slot.

Reply:
do we not validate that extraneous fields aren't present?

Reply:
Depends on whether undefined extensions are supported or not.

Without support for undefined extensions, non-standard slots are _ignored_ (possibly with a warning, though that’s not mandatory). That’s what would happen with SSSOM-Py (which does not support extensions at all), and with SSSOM-Java by default (where support for undefined extensions must be explicitly enabled).

With support for undefined extensions, any non-standard slot is treated as an undefined extension.

In any case, the point here is that if you want a SSSOM 1.1 slot (like this `mapping_confidence_set`) to be recognised as a valid slot instead of being ignored or treated as an undefined extension, the set _must_ declares itself as a 1.1 set, by means of the `sssom_version="1.1"` slot.

Reply:
Thank you @gouttegd I added the slot information!